core/src/sharedTestFunctions/java/org/kiwix/sharedFunctions/TestModelFunctions.kt

```diff
@@ -164,3 +165,14 @@ fun libraryNetworkEntity(books: List<Book> = emptyList()) = LibraryNetworkEntity
 
 fun recentSearchEntity(id: Long = 0L, searchTerm: String = "", zimId: String = "") =
   RecentSearchEntity(id, searchTerm, zimId)
+
+fun bookmarkItem(
+  databaseId: Long = 0L,
+  zimId: String = "",
+  zimName: String = "",
+  zimFilePath: String?,
```

Comment:
to keep with the style of the rest of these functions you should supply default parameters for all fields of the object